app/models/ability.rb

```diff
@@ -17,11 +17,11 @@ def custom_permissions
     # end
     ezid_shoulder = Rails.application.secrets['doi']['default_shoulder']
     cannot [:update, :destroy], ::Collection do |c|
-      c.identifier.any? {|id| id.start_with?(ezid_shoulder)}
+      !c.identifier.blank?
```

Comment:
What if the identifiers being edited as [""] or [nil], .blank? returns false. Maybe it's better to check using .any? for each of the identifiers.